File: testprojects/csharp/benchmark/Technology/Direct/Subclass/CallInstanceSubClassDOA.cs
using Technology.Direct.Dao;

namespace Technology.Direct.Subclass
{
	public class CallInstanceSubClassDOA : CallInstanceSuperClassDAO
	{
		public CallInstanceSubClassDOA()
		{
		}

		public CallInstanceSuperClassDAO MethodSuperClassInstance;

		public virtual void Test()
		{
			MethodSuperClassInstance.MethodOnSuperClass();
		}
	}
}

File: testprojects/java/benchmark/accuracy-test-2013-04-17/technology/direct/subclass/CallInstanceSubClassDOA.java
package technology.direct.subclass;

import technology.direct.dao.CallInstanceSuperClassDAO;

public class CallInstanceSubClassDOA extends CallInstanceSuperClassDAO {
	
	public CallInstanceSubClassDOA() {
		};
	
	public CallInstanceSuperClassDAO MethodSuperClassInstance;
	
	public void test(){
		MethodSuperClassInstance.MethodOnSuperClass();
	}
}
